model/src/main/scala/ch.epfl.scala.index.model/release/ScalaTarget.scala

```diff
@@ -120,12 +120,12 @@ object ScalaJs {
 
 object ScalaNative {
   val `0.3`: BinaryVersion = MinorBinary(0, 3)
-  val `0.4.0-M2`: BinaryVersion =
-    release.PreReleaseBinary(0, 4, Some(0), Milestone(2))
+  val `0.4`: BinaryVersion = MinorBinary(0, 4)
 
-  def isValid(version: BinaryVersion): Boolean = {
-    version == `0.3` || version >= `0.4.0-M2`
```

Comment:
I would have expected `0.4` to be greater than `0.4.0-M2` but apparently it is not:

```scala
assert(MinorBinary(0, 4) > PreReleaseBinary(0, 4, None, Milestone(2))) // success
assert(MinorBinary(0, 4) > PreReleaseBinary(0, 4, Some(0), Milestone(2))) // failure
```
Looks like a bug to me... Would you try to fix it?

Reply:
Sure, not a problem